src/Content/Post/Factory/PostMedia.php

```diff
@@ -48,24 +49,24 @@ public function createFromTableRow(array $row)
 	{
 		return new Entity\PostMedia(
 			$row['uri-id'],
-			$row['url'] ? new Uri($row['url']) : null,
+			UtilNetwork::createUriFromString($row['url']),
```

Comment:
Since you 're catching the Exception in the repository, you don't need to use `createUriFromString` here anymore.
```suggestion
			new Uri($row['url']),
```

Reply:
This would behave differently when `$row['url']` was an empty string. In that case the `new` doesn't throw an error. But we really should do so for that case.